main.js

```diff
@@ -0,0 +1,41 @@
+render = Renderer()
+brain = Brainer()
```

Comment:
When declaring a new variable you should declare it with _const_ , otherwise, the variable will be defined as "_var_"